source/_components/switch.xiaomi_vacuum

```diff
@@ -0,0 +1,36 @@
+The `xiaomi_vacuum`switch platform allows you to control the state of your [Xiaomi Mi Robot Vacuum](http://www.mi.com/roomrobot/). 
+Current supported features are start and stop (go to dock).
+
+{% linkable_title Getting started %}
+
+To be able to communicate with the vacuum one needs to have its IP address as well as an encryption token.
+This token is only attainable before the device has been connected over the app to your local wifi.
+
+In order to fetch the token, reset the robot, connect to its the network its announcing (rockrobo-XXXX)
+and run the following command:
```

Comment:
I'm wondering if this part is too vague (for users who do not know how to access the virtualenv/environment of homeassistant), but can't directly think of how to write it out in a nicer manner.